scout/server/blueprints/variant/templates/variant/sv-variant.html

```diff
@@ -130,10 +130,10 @@
               {{ variant.chromosome }}:{{ variant.position }}
               {% if variant.chromosome == "MT" and case.mt_bams %}
                 - Alignment:
-                <a class="btn btn-outline-secondary btn-sm" href="{{ url_for('alignviewers.igv', sample=case.sample_names, build=case.genome_build, bam=case.mt_bams, bai=case.mt_bais, contig=variant.chromosome, start=(variant.position - 500), stop=(variant.position + 500)) }}" target="_blank">IGV viewer</a>
+                <a class="btn btn-outline-secondary btn-sm" href="{{ url_for('alignviewers.igv', sample=case.sample_names, build=case.genome_build, bam=case.mt_bams, bai=case.mt_bais, rhocall_bed=case.rhocall_beds, rhocall_wig=case.rhocall_wigs, tiddit_coverage_wig=tiddit_coverage_wigs, upd_regions_bed=case.upd_regions_beds, upd_sites_bed=case.upd_sites_beds, contig=variant.chromosome, start=(variant.position - 500), stop=(variant.position + 500)) }}" target="_blank">IGV viewer</a>
```

Comment:
It's not mandatory, but if we pass a ton of kind of identical things to each of these, couldn't we make jinja accept some higher order structure with them somehow?

Reply:
Yeah, it's getting out of hand. I suggest making it a separate issue.